docs/solana/migration/ledger.mdx

```diff
@@ -20,10 +20,11 @@ these steps:
 
 1. Open Ledger Live.
 2. Go to Settings (top right corner)
-3. Install the Helium-Solana app as you normally would.
-4. Click `Experimental Features`
-5. Enable `Developer Mode`
-6. Change `My ledger provider` to `4`
+3. Click `Experimental Features`
+4. Enable `Developer Mode`
+5. Change `My ledger provider` to `4`
+6. Install the Helium-Solana app as you normally would,
+   [following the instructions here](/wallets/ledger).
```

Comment:
Can you confirm that this is the correct change? It was mixed in with this branch.